src/UI/Implementation/Component/Triggerer.php

```diff
@@ -8,6 +8,8 @@
  *
  * Provides helper methods and default implementation for components acting as triggerer
  *
+ * TODO: This is missing tests!
```

Comment:
Can you provide those? If not, who is supposed to provide those?

Reply:
In general yes, currently I have no time for that.

The complete trait currently is missing tests, we should have requested those when that trait was initially implemented. So atm I guess no one is supposed to provide them.